docs/reference/esql/esql-language.asciidoc

```diff
@@ -20,7 +19,6 @@ include::esql-syntax.asciidoc[]
 include::esql-commands.asciidoc[]
 include::esql-functions-operators.asciidoc[]
 include::metadata-fields.asciidoc[]
-include::esql-index-options.asciidoc[]
```

Comment:
Should we go ahead and remove the [actual file](https://github.com/elastic/elasticsearch/blob/2730fd390301d30bc3e7c521437872e755a3494e/docs/reference/esql/esql-index-options.asciidoc) too? 

(but it's also fine to do that in a later separate PR -- removing the `include` line does the trick)

Reply:
Thanks, removed.